### ISSUE TITLE
Implement favourites on NTP similar to iOS

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#7258]
Android should support default favourites that can be customised instead of top tiles based on browsing history. This would bridge gap between iOS and Android and also allows to still maintain more favourites with new tab image

cc: @jamesmudgett @deeppandya  

@SergeyZhukovsky @anthonypkeane if possible we should get this in first core build for Android

**Comment by VideoPlayerCode** [comment]
Same GUI design as top sites.

With many potential ways to add sites:

- Tap-and-hold on blanks space bringing up a menu for Add bookmark, which let's you type a name and URL. I like this option the least.
- Visit a site and when adding to bookmark have a "add to dashboard" option instead of adding to the regular shortcut folders. 
- Have a "Dashboard Favorites" auto-generated folder inside the existing bookmarks editor, and let users edit and move bookmarks into/out of that group.
- Have the current bookmark system but each bookmark has a "Show on Dashboard" checkbox on the Edit page. 
- Make users add bookmarks to the bookmark menu as usual (as it is done today), but Afterwards they can go to their Dashboard and tap-and-hold on a blank space and "Assign Bookmark Here" and pick one of their bookmarks to display at that spot. These would be links, so if the user later edits a bookmark the dashboard automatically shows the updated info. Also, whenever the user creates new bookmarks, have a checkmark for "Add Dashboard Shortcut" which immediately stores the created bookmark in the first available dashboard slot.

I prefer the latter solution. Seems the most polished to me.

**Comment by eljuno** [comment]
+1 from Community https://community.brave.com/t/creating-fixed-bookmarks-on-the-startup-page/140139?u=eljuno

**Comment by JakobSievers** [comment]
Yes! I am eagerly looking for this as well. The absence of something like this is the main reason I keep gravitating towards the default Samsung browser on my Galaxy S8. It is just such a nice, quick and seamless experience.

**Comment by Penguinjumper** [comment]
Still nothing after 5 years? All the other Browsers (and versions of brave on other OS) have it!

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/33845 - we did make some progress here (should be a branch w/ code) we just got caught up with implementation details when nearing completion and then lost track of the work. Thanks for your patience
